content/docs/common-areas/cdn.md

```diff
@@ -0,0 +1,41 @@
+---
+title: CDN (Content Delivery Network)
+weight: 5
+description: A content delivery network (CDN) is a group of geographically distributed servers that speed up the delivery of web content by bringing it closer to where users are. Data centers across the globe use caching, a process that temporarily stores copies of files, so that you can access internet content from a web-enabled device or browser more quickly through a server near you. CDNs cache content like web pages, images, and video in proxy servers near to your physical location. This allows you to do things like watch a movie, download software, check your bank balance, post on social media, or make purchases, without having to wait for content to load.
+---
+
+# CDN (Content Delivery Network)
+
+A content delivery network (CDN) is a geographically distributed group of servers that caches content close to end users. A CDN allows for the quick transfer of assets needed for loading Internet content, including HTML pages, JavaScript files, stylesheets, images, and videos. The popularity of CDN services continues to grow, and today the majority of web traffic is served through CDNs, including traffic from major sites like Facebook, Netflix, and Amazon.
+
+## Is a CDN the same as a web host?
+
+While a CDN does not host content and can’t replace the need for proper web hosting, it does help cache content at the network edge, which improves website performance. Many websites struggle to have their performance needs met by traditional hosting services, which is why they opt for CDNs.
+
+By utilizing caching to reduce hosting bandwidth, helping to prevent interruptions in service, and improving security, CDNs are a popular choice to relieve some of the major pain points that come with traditional web hosting.
+
+## Benefits of using a CDN
+
+The benefits of using a CDN depend on the size and needs of an Internet property. The following are the common benefits:
+
+- Improving website load times: web site’s loading time gets shorter because a CDN reuses the sessions previously opened by visitors. In such a way that when a user wants to access the website, it will load quickly and no need to wait to get the full view of the website. In case the caching content is lost they can be recuperated by the interconnection with other CDN servers.
+- Another major concern for websites is the consumption of bandwidth for hosting. The quantity of data that an origin server needs to provide can be reduced by CDNs through caching and other improvements, which lowers hosting costs for website owners.
```

Comment:
This feels a little inaccurate. Sure, hosting costs go down but then you pay for the CDN which could end up being more expensive. CDNs are not designed to save cost, they are for performance, and that makes this point unnecessarily misleading.

Specifically, when you say that "The quantity of data that an origin server needs to provide can be reduced", that is definitely misleading. The origin server still needs the data. The only quantity that is reduced is the transfer out size.